pages/settings/SettingsDisruptions.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+
+import { Column } from "@/components/core/Column";
+import { Text } from "@/components/core/Text";
+import { Spacer } from "@/components/core/Spacer";
+import {
+  filterableDisruptionCategories,
+  FilterableDisruptionCategory,
+  Settings,
+} from "@/shared/settings";
+
+export type DisruptionSettingsProps = {
+  settings: Settings;
+  setSettings: (settings: Settings) => void;
+};
+
+const allCategories = ["essential", ...filterableDisruptionCategories] as const;
+
+const formattedCategories: Record<
+  (typeof allCategories)[number],
+  { name: string; description?: string; disabled?: boolean }
+> = {
+  essential: {
+    name: "Essential disruptions",
+    description: "Buses replace trains, line closures",
+    disabled: true,
+  },
+  "station-closures": {
+    name: "Station closures",
+  },
+  cancellations: {
+    name: "Cancellations",
+  },
+  delays: {
+    name: "Delays",
+  },
+  "car-park-closures": {
+    name: "Car park closures",
+  },
+  accessibility: {
+    name: "Accessibility",
+    description: "Lift outages, stair-only access, etc.",
+  },
+};
+
+export function SettingsDisruptions({
+  settings,
+  setSettings,
+}: DisruptionSettingsProps) {
+  function toggleHiddenCategory(category: FilterableDisruptionCategory) {
+    if (settings.hiddenCategories.includes(category)) {
+      setSettings(settings.withoutHiddenCategory(category));
+    } else {
+      setSettings(settings.withHiddenCategory(category));
+    }
+  }
+
+  return (
+    <Column>
+      <Text style="custom" className="text-lg font-bold">
+        Disruptions to show
+      </Text>
+      <Spacer h="2" />
+
+      <Column className="gap-2">
+        {allCategories.map((category) => (
+          <label
+            key={category}
+            className="flex cursor-pointer items-center justify-between hover:bg-gray-200 dark:hover:bg-gray-600"
+          >
+            <input
+              type="checkbox"
+              value={category}
+              autoComplete="off"
+              className="peer sr-only"
+              checked={
+                (settings.hiddenCategories as string[]).includes(category) ||
+                formattedCategories[category].disabled === true
+              }
+              disabled={formattedCategories[category].disabled}
+              onChange={
+                category !== "essential"
+                  ? () => toggleHiddenCategory(category)
+                  : undefined
+              }
+            />
+            <Column>
+              {formattedCategories[category].name}
```

Comment:
```suggestion
              <Text>{formattedCategories[category].name}</Text>
```
Best to always wrap text in `<Text>`, since `<Text>` has some magic to fix line spacing.

(Also spotted [here](https://github.com/dan-schel/train-disruptions/blob/1957f716bc3d50cd3201914330b8f1bfd5511b4c/pages/settings/SettingsTheme.tsx#L53) and [here](https://github.com/dan-schel/train-disruptions/blob/1957f716bc3d50cd3201914330b8f1bfd5511b4c/pages/settings/SettingsHome.tsx#L52).)

Reply:
Yep, will update them.